### ISSUE TITLE
An error of  check_loaded_data function

### ISSUE THREAD

**Issue by xueyouxz** [vita-epfl/UniTraj#45]
In the `check_loaded_data `function, there is a line of code:
`agents = np.concatenate([data['obj_trajs'][..., :2], data['obj_trajs_future_state'][..., :2]], axis=-3)`
I’m not sure what this line does, but this function is called in `base_dataset.py`, as shown in the following form:
```
def draw_figures(cfg):
   ...
    for n, data in tqdm(enumerate(train_loader)):
        input_dict = data['input_dict']
        for i in range(data['batch_size']):
              plt = check_loaded_data(data['input_dict'], i)
              ...
```
An error occurs when calling` check_loaded_data`, and the external parameter passed to `data `is `i`. I am currently unsure about the role and meaning of this parameter. Could you help clarify this?

**Comment by Alan-LanFeng** [comment]
It represents visualizing the ith sample in the batch
